cinema_service/urls.py

```diff
@@ -4,5 +4,6 @@
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("api/cinema/", include("cinema.urls", namespace="cinema")),
+    path("api/user/", include("user.urls", namespace="user")),
```

Comment:
Ensure that both `cinema.urls` and `user.urls` have `app_name` defined to work correctly with namespaces. This is necessary for the `namespace` argument to function properly.